spec/gyro/xcdatamodel_spec.rb

```diff
@@ -24,10 +24,10 @@ module Gyro
         .to raise_error 'Unable to find contents of xcdatamodel'
     end
 
-    it 'check raising relationship error' do
-      xcdatamodel_dir = fixture('xcdatamodel', 'error_relationship.xcdatamodel')
+    it 'check raise an error for xcdatamodeld' do
+      xcdatamodel_dir = fixture('xcdatamodel', 'Model.xcdatamodeld')
       expect { Parser::XCDataModel::XCDataModel.new(xcdatamodel_dir) }
-        .to raise_error 'The relationship "user" from "FidelityCard" is wrong - please fix it'
```

Comment:
The new spec is good, but why did the old `"check raising relationship error"` spec disappear?

Reply:
My bad I will fix this